src/main/kotlin/no/nav/familie/tilbake/faktaomfeilutbetaling/domain/HendelsestypePerYtelsestype.kt

```diff
@@ -13,13 +13,25 @@ object HendelsestypePerYtelsestype {
                                                                  Hendelsestype.DELT_BOSTED,
                                                                  Hendelsestype.BARNS_ALDER),
                                  Ytelsestype.OVERGANGSSTØNAD to setOf(Hendelsestype.ANNET,
-                                                                      Hendelsestype.BOR_MED_SØKER,
+                                                                      Hendelsestype.MEDLEMSKAP,
+                                                                      Hendelsestype.OPPHOLD_I_NORGE,
+                                                                      Hendelsestype.ENSLIG_FORSØRGER,
+                                                                      Hendelsestype.OVERGANGSSTØNAD,
+                                                                      Hendelsestype.YRKESRETTET_AKTIVITET,
+                                                                      Hendelsestype.STØNADSPERIODE,
+                                                                      Hendelsestype.INNTEKT,
+                                                                      Hendelsestype.PENSJONSYTELSER,
                                                                       Hendelsestype.DØDSFALL),
```

Comment:
Tror ikkje me kan gjenbruke DØDSFALL for EF-ytelsane. Dei skal ha forskjellig tekst i frontend, og kan ikkje skille på ytelse/fagssystem for å vise tekst.
Bør skifte til EF_DØDSFALL?

Reply:
Var litt rask der. Dei har faktisk tenkt på det, og ikkje inkludert paragraf for EF-ytelsane.